open-metadata-implementation/repository-services/repository-services-implementation/src/test/java/org/odpi/openmetadata/repositoryservices/localrepository/repositorycontentmanager/OMRSRepositoryContentManagerTest.java

```diff
@@ -0,0 +1,52 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.repositoryservices.localrepository.repositorycontentmanager;
+
+import org.odpi.openmetadata.frameworks.auditlog.AuditLog;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceStatus;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceType;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.typedefs.EntityDef;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.typedefs.TypeDefCategory;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.TypeErrorException;
+import org.testng.annotations.Test;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
+import static org.mockito.Mockito.*;
+import static org.testng.Assert.assertEquals;
+
+
+public class OMRSRepositoryContentManagerTest {
+
+
+
```

Comment:
Why do we need Mockito for this test?

Reply:
Mockito was initially used to quickly create a no-op `AuditLog` without cluttering the test case with the chain of`OMRSAuditLogDestination` to `OMRSAuditLogStore`  object instantiation. After your comment, I looked into `AuditLog` implementation and found that the same behaviour can be achieved by passing a null `AuditLogDestination` and removed Mockito.